packages/mobile/test/schemas.ts

```diff
@@ -702,6 +702,13 @@ export const v16Schema = {
     exchangeRates: null,
     exchangeRatePair: undefined,
   },
+  verify: {
+    ...v15Schema.verify,
+    komenciConfig: {
+      useLightProxy: false,
+      allowedDeployers: [],
+    },
+  },
```

Comment:
We should add this to a new schema.
Since the last public release already used schema v16, it's safer to make a new one as we might forget about this change later on.

Reply:
By the way I missed this in https://github.com/valora-inc/wallet/commit/49d0e31b7621683d716d0ae7613acae3e7c9f273#diff-a0850796be5a07a6363631e739d56ea69fa51c26852655610c6f7650285b9a2e and the new schema should have been added there.

Reply:
Isn't the new schema's use for new migrations? If I add this with no migration both will end up desynced, no?

Reply:
A v17 was introduced in another PR with a migration so I moved this there instead. I don't think I understand the logic for this versioning though :/